ui/src/main/java/com/kotcrab/vis/ui/widget/VisImageTextButton.java

```diff
@@ -108,6 +111,29 @@ public boolean touchDown (InputEvent event, float x, float y, int pointer, int b
 		});
 	}
 
+	private void addActorsBasedOnOrientation() {
+		switch (orientation) {
+			case text_right:
+				add(image);
+				add(label);
+				break;
+			case text_left:
+				add(label);
+				add(image);
+				break;
+			case text_top:
+				add(label);
+				row();
+				add(image);
+				break;
+			case text_bottom:
+				add(image);
+				row();
+				add(label);
```

Comment:
Assigning and providing access to the image and label cells might ease customization.

Reply:
Users can get at the cells via `getImageCell()` and `getLabelCell()` so caching those in member variables for performance purposes can be added later if needed.